### PR TITLE
Make sure './configure && make all install' work.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**

### CODE REVIEW

**Comment by quixotique** [comment]
Hi Peter,

Thanks for the patch.  Before I pull it, I'd like to clarify a copyright issue.

The Serval Project normally requires that all contributors sign a copyright assignment agreement (see http://developer.servalproject.org/dokuwiki/doku.php?id=content:dev:).  We do this so that in future we could dual license the Serval source code to GPL-hostile platforms, eg iOS and Windows Mobile.  Although dual licensing for closed platforms would go against the principles of free software, our fundamental humanitarian objective means we must retain it as an option.

However, I would venture that the changes in your patch are so small and standard that they are not copyrightable, so I could pull them straight away.  (By “standard” I mean that anyone seeking to implement `make install` within the autotools framework would come up with exactly the same lines in Makefile.ini.)

Would you agree?

**Comment by petterreinholdtsen** [comment]
[Andrew Bettison]

> Hi Peter,

Hi.  (Peter is my cousin.  I am Petter. :-)

> Thanks for the patch.  Before I pull it, I'd like to clarify a
> copyright issue.

Thank you for bringing 

> The Serval Project normally requires that all contributors sign a
> copyright assignment agreement (see
> http://developer.servalproject.org/dokuwiki/doku.php?id=content:dev:).
> We do this so that in future we could dual license the Serval source
> code to GPL-hostile platforms, eg iOS and Windows Mobile.  Although
> dual licensing for closed platforms would go against the principles
> of free software, our fundamental humanitarian objective means we
> must retain it as an option.

In general, I am not very keep in copyright assignment agreement with
my free software development, as I prefer to work on projects where
every contributor have equal rights.  And such requirement will block
people from contributors where their employer is unwilling to approve
of the transwer.  Another way for your stated goal of having the code
available on GPL-hostile platforms would be to use a BSD like license,
so I fail to see why copyright assignment is the correct solution.  In
addition, the copyright assignment agreement is very broad,
transfering quite a lot more than as I can tell, would be required to
use the code on GPL-hostile platforms.

In short, when I donate my time and knowledge, I do not like getting a
legal agreement pushed in my face and asked to sign away rights I
otherwise would have.

> However, I would venture that the changes in your patch are so small
> and standard that they are not copyrightable, so I could pull them
> straight away.  (By “standard” I mean that anyone seeking to
> implement `make install` within the autotools framework would come
> up with exactly the same lines in Makefile.ini.)
> 
> Would you agree?

Yes.  This patch is a trivial implementation of the 'make install'
pattern used by any package using autoconf.

## 

Happy hacking
Petter Reinholdtsen

**Comment by gardners** [comment]
Petter,
As mentioned we are happy to accept contributions given on a dual-licensed BSD+GPL basis.
Paul.

**Comment by quixotique** [comment]
Pulled manually.
